main.go

```diff
@@ -296,12 +299,11 @@ func beforeConnectAction(ctx *cli.Context) error {
 
 	// When machine-readable format is used, then additional requirements have to be met
 	if uiSettings.isMachineReadable {
-		if username != "" {
-			if password == "" {
-				return fmt.Errorf("--password is required, when --username and machine-readable format are used")
-			}
+		if username == "" || password == "" {
+			return fmt.Errorf("--username/--password or --organization/--activation-key are required, when machine-readable format is used")
```

Comment:
```suggestion
			return fmt.Errorf("--username/--password or --organization/--activation-key are required when a machine-readable format is used")
```

Minor grammatical changes to error message.